include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -463,7 +463,7 @@ namespace realization {
                 if (directory != nullptr) {
                     bool match;
                     while ((entry = readdir(directory))) {
-                        match = std::regex_match(entry->d_name, pattern);
+                        match = boost::regex_match(entry->d_name, pattern);
                         if( match ) {
```

Comment:
```suggestion
                        if (filepattern == entry->d_name || boost::regex_match(entry->d_name, pattern)) {
```

Reply:
I've tested this and it works, it's also inspired me to see if there's a way to filter the directory with regex, then loop over the matches, rather than having to check each file one at a time. It looks possible with boost::filesystem, but that might be a bit too cumbersome for this